theme/js/dataset-recommendations.js

```diff
@@ -20,6 +26,10 @@ function addRecos(recos) {
         recoChildContainer.appendChild(recoChildEmbed);
         recoContainer.appendChild(recoChildContainer);
     });
+
+    if (!(typeof _paq == 'undefined')) {
```

Comment:
Proposal, not sure if better (I don't like this syntax).

```
const paq = _paq || {};
if (paq.push) { paq.push [...] }
```

Reply:
:joy: 
Just published mine below. I don't like this too

Reply:
@abulte your proposal was a bit buggy for some obscure reason of direction of execution.